### PR TITLE
Support translation (l10n)

### PR DESCRIPTION
**Pull request by outscale-mdr**
Use of the new APi of vscode l10n acoming with v1.73 vscode version (see [here](https://github.com/microsoft/vscode-l10n#vscodel10nt)) therefore we must increase minimal version of the plugin.

### CODE REVIEW

**Review by jerome-jutteau — CHANGES_REQUESTED** [review]

**Comment by outscale-mdr** [comment]
I've tried to tokenize messages but vscode does not support it. Indeed, it takes the token as the english language (see https://github.com/microsoft/vscode/issues/166939)

File: l10n/bundle.l10n.fr.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "Error while reading {0}: {1}": "Erreur lors de la récupération de {0}: {1}",
+    "Add a profile": "Ajouter un profile",
+    "Choose a unique name for the profile": "Choissiser un nom unique pour le profile",
+    "The name is not unique": "Le nom n'est pas unique",
+    "Cannot be empty": "Ne peut être vide",
+    "Enter your Access Key": "Entrez l'Access Key",
+    "Enter your Secret Key": "Entrez la Secret Key",
+    "Pick a region": "Choissisez une region",
+    "Enter the host": "Entrez l'hôte",
+    "Use HTTPS ?": "Utilisez HTTPS ?",
```
Comment (jerome-jutteau) [inline l10n/bundle.l10n.fr.json:38]: Pour la question, j'utiliserai le verbe à l'infinitif: utiliser, comme dans "voulez-vous utiliser HTTPS ?"

File: l10n/bundle.l10n.fr.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "Error while deleting the subresource of {0}: {1}": "Erreur lors de la suppression de la sous-ressource {0}: {1}",
+    "Deletion of subresource cancelled": "Suppression de la sous-ressource annulée",
+    "The subresource is incomplete": "La sous-ressource n'est pas complète",
+    "The resource has no subresources": "La ressource n'a pas de sous-ressource",
+    "The resource {0} has been unlinked": "La ressource {0} a été désassociée",
+    "Error while unlinking the resource {0}: {1}": "Erreur lors de la désassocation de la ressource {0}: {1}",
+    "Do you want to unlink the resource {0} ?": "Voulez-vous désassocier la ressource {0} ?",
+    "Unlink cancelled": "Désassocation annulée",
+    "Error while starting the resource {0}: {1}": "Erreur lors du démarrage de la ressource {0}: {1}",
+    "The resource {0} has been started": "La ressource {0} a été démarré",
```
Comment (jerome-jutteau) [inline l10n/bundle.l10n.fr.json:18]: démarrée ?

File: l10n/bundle.l10n.fr.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    "Error while stopping the resource {0}: {1}": "Erreur lors de l'arrêt de la ressource {0}: {1}",
+    "The resource {0} has been stopped": "La ressource {0} a été arrêté",
+    "Do you want to stop the resource {0} ?": "Voulez-vous arrêter la ressource {0} ?",
+    "Yes": "Oui",
+    "No": "Non",
+    "No config file found": "Aucun fichier de configuration trouvé",
+    "Copied to clipboard": "Copié dans le presse-papiers",
+    "Error while reading {0}: {1}": "Erreur lors de la récupération de {0}: {1}",
+    "Add a profile": "Ajouter un profile",
+    "Choose a unique name for the profile": "Choissiser un nom unique pour le profile",
```
Comment (jerome-jutteau) [inline l10n/bundle.l10n.fr.json:31]: choisissez ?

**Review by jerome-jutteau — CHANGES_REQUESTED** [review]

**Review by jerome-jutteau — APPROVED** [review]

**Pull request merged**